README.md

```diff
@@ -16,28 +16,35 @@ Klingt das interessant für Dich? Dann mach mit, herzliche Einladung!
 
+Die Umschaltung zwischen den beiden Hardware-Versionen geschieht über die Datei **userdefines.h** und (bei platformio) zusätzlich auch über die Datei **platformio.ini**. Beide Dateien sind **nicht** auf GitHub enthalten, sondern jeweils nur die Muster **userdefines-example.h** und **platformio-example.ini**. Diese beiden Dateien müssen also kopiert werden: **userdefines-example.h** nach **userdefines.h** und **platformio-example.ini** nach **platformio.ini**.     
+Änderungen und Einstellungen dann **nur** in den neuen Dateien machen!
+
  * **Heltec WiFi Kit 32**  
 Diese MCU hat ein großes Display und WiFi. Auf dem Board wird dieser Baustein in die längeren Buchsenleisten gesteckt. Für die Arduino-IDE ist als Board der **Heltec WiFi Kit 32** einzustellen, in der Datei *userdefs.h* muss **#define CPU WIFI** entkommentiert werden (die anderen CPU-defines werden auskommentiert).   
-Für Platfomio ist in *platformio.ini* ganz oben **default_envs = stick** einzustellen. In *userdefs.h* ist (wie bei Arduino-IDE) das **#define CPU WIFI** zu entkommentieren.
+Für Platfomio ist in *platformio.ini* ganz oben **default_envs = wifi** einzustellen. In *userdefs.h* ist (wie bei Arduino-IDE) das **#define CPU WIFI** zu entkommentieren.
+
  * **Heltec Wireless Stick**  
 Diese Board hat ein sehr kleines Display, dafür aber zusätzlich zu WiFi noch LoRa. Es wird in die kürzeren Buchsenleisten gesteckt. Für die Arduino-IDE muss als Board **Heltec Wireless Stick** eingestellt werden. In *userdefs.h* wird nun das **#define CPU STICK** entkommentiert, die anderen mit Kommentarzeichen versehen).  
-Für Platfomio ist in *platformio.ini* ganz oben **default_envs = wifi** einzustellen. In *userdefs.h* gleich wie bei der Arduino-IDE.  
-Zusätzlich muss in *userdefs.h* eingestellt werden, dass zu **TTN** gesendet werden soll. Dazu **#define SENDT2ORA 1** anstelle von 0 einstellen. Die anderen beiden können (SEND2MADAVI und SEND2LUFTDATEN) können entweder auf 1 bleiben (dann wird auch dahin gesendet) oder auf 0 gesetzt werden (siehe weiter unten).  
+Für Platfomio ist *platformio.ini*  ganz oben **default_envs = stick** einzustellen. In *userdefs.h* gleich wie bei der Arduino-IDE.  
+Zusätzlich muss in *userdefs.h* eingestellt werden, dass zu **TTN** gesendet werden soll. Dazu **#define SENDT2ORA 1** anstelle von 0 einstellen. Die anderen beiden können (SEND2MADAVI und SEND2LUFTDATEN) entweder auf 1 bleiben (dann wird auch dahin gesendet) oder auf 0 gesetzt werden (siehe weiter unten).  
 Die **LoRa**-Credentials werden in der Datei *lorawan.cpp* ab Zeile 65 eingetragen.
 
 
 Als externe Libraries werden benötigt:   
 
- * U8g2 von Oliver, aktuelle Version 2.16.4  
+ * U8g2 von Oliver, aktuelle Version 2.16.14  
  * Adafruit BME280 Library Version 1.0.7  
  * Adafruit Unified Sensor Version 1.02  
+
+für LoRa zusätzlich:
+ * MCCI LoRaWAN LMIC library Version 2.3.2
```

Comment:
after writing my bug report about this, i discovered that there is a lmic.h in our github repo, see #17.

so:
- should this lib be added via the arduino library manager (and do we need to have lmic.h etc. in our repo in that case?)
- or should the included header / lib be used (how?)

Reply:
(in case the library files should be removed from git, please do a separate commit / PR for that)